js/templates/mobile-nav.hbs

```diff
@@ -11,7 +11,6 @@
         <label class="t-sans" for="menu-search">Search for candidates and committees</label>
         <div class="combo--search--mini">
           <input id="menu-search" name="search" class="combo__input js-menu-search" type="text"
-          placeholder="Search for candidates"
```

Comment:
Oh so interesting! Actually, only the mobile nav says "Search for candidates and committees" the desktop nav says "Search for candidates" (and omits committees, which people can't search for now. I'm going to change the mobile navigation to match the desktop (i.e., leave off the "and committees" part)